### ISSUE TITLE
Additional 48in24 exercises

### ISSUE THREAD

**Issue by BNAndras** [exercism/erlang#592]
@exercism/erlang, according to https://exercism.org/challenges/48in24/implementation_status/erlang, here are five missing exercises for this track that the LFE track has added. Should I go ahead and add them?

- [x] acronym #597
- [x] dnd-character #611 (Maartz)
- [x] game-of-life #613 
- [x] yacht #602
- [x] kindergarten-garden #612

**Comment by NobbZ** [comment]
Yes, feel free to implement them, I am happy to review and merge them.

**Comment by BNAndras** [comment]
Thanks, I'll work on these later this month / next month.

**Comment by Maartz** [comment]
A dnd-character PR in #611 :)

**Comment by BNAndras** [comment]
I hallucinated Pig Latin being implemented on the LFE track. I'm going to substitute in https://exercism.org/exercises/game-of-life instead.
